backend/src/main/java/com/eshka/dto/response/TeamFullResponse.java

```diff
@@ -0,0 +1,35 @@
+package com.eshka.dto.response;
+
+import com.eshka.entity.Team;
+import com.eshka.mapper.SubjectMapper;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class TeamFullResponse {
+    private long id;
+    private String name;
+    private String description;
+    private List<SubjectResponse> subjects;
+    private List<UserShortResponse> members;
+
+    public static TeamFullResponse fromTeam(Team team) {
+        return new TeamFullResponse(
+                team.getId(),
+                team.getName(),
+                team.getDescription(),
+                team.getSubjects().stream()
```

Comment:
unacceptable for entity class. This Business Logic must be in Services or mapper